### PR TITLE
Integrated XToken Pallet in runtime

### PR DESCRIPTION
**Pull request by zktony**
* Integrated XToken Pallet in runtime.
* Added two new extrinsic in xcm_handler.
Will close #6

### CODE REVIEW

File: xcm-simulator/src/lib.rs [hunk 1]
```diff
@@ -144,10 +146,57 @@ mod tests {
 
 		ParaA::execute_with(|| {
 			use parachain::{RuntimeEvent, System};
+			println!("All events {:?}", System::events());
```
Comment (serhii-temchenko) [inline xcm-simulator/src/lib.rs:149]: Please remove

File: xcm-simulator/src/lib.rs [hunk 2]
```diff
@@ -144,10 +146,57 @@ mod tests {
...
 				r.event,
 				RuntimeEvent::XcmHandler(xcm_handler::Event::AssetDeposited { .. })
 			)));
 		});
 	}
+
+	#[test]
+	fn test_withdraw_from_parachain_to_relay_chain() {
+		MockNet::reset();
+		// TODO for R
```
Comment (serhii-temchenko) [inline xcm-simulator/src/lib.rs:160]: Please extend todo info

File: xcm-simulator/src/lib.rs [hunk 3]
```diff
@@ -144,10 +146,57 @@ mod tests {
...
+		Relay::execute_with(|| {
+			//pallet_balances::Pallet::<parachain::Runtime>::deposit_creating(&para_account_id(1), 1_000_000_000_000);
```
Comment (serhii-temchenko) [inline xcm-simulator/src/lib.rs:162]: Please remove

File: xcm-simulator/src/parachain.rs [hunk 4]
```diff
@@ -232,7 +199,7 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 					}
 			)
 		}) {
-			return Err(()) // Deny
+			//return Err(()) // Deny
```
Comment (serhii-temchenko) [inline xcm-simulator/src/parachain.rs:202]: Why is this commented out?

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: xcm-simulator/src/parachain.rs (hunk 4 above)
Comment (zktony) [reply xcm-simulator/src/parachain.rs:202]: done

**Review by serhii-temchenko — APPROVED** [review]

**Pull request merged**